v0/destinations/campaign_manager/transform.js

```diff
@@ -42,7 +62,9 @@ function buildResponse(
     requestType === "batchinsert"
       ? "dfareporting#conversionsBatchInsertRequest"
       : "dfareporting#conversionsBatchUpdateRequest";
-  response.body.JSON.encryptionInfo = encryptionInfo || {};
+  if (!isEmptyObject(encryptionInfo)) {
```

Comment:
```suggestion
  if (!isDefinedAndNotNull(encryptionInfo)) {
```

Reply:
this was not working for empty object( {})